dp-integrity-checker.nomad

```diff
@@ -0,0 +1,69 @@
+job "dp-integrity-checker" {
+  datacenters = ["eu-west-1"]
+  region      = "eu"
+  type        = "batch"
+
+  periodic {
+    cron             = "0 0 1 * * 1-5 *"
+    time_zone = "UTC"
+    prohibit_overlap = true
+  }
+
+  update {
+    stagger          = "60s"
+    min_healthy_time = "30s"
+    healthy_deadline = "2m"
+    max_parallel     = 1
+    auto_revert      = true
+  }
+
```

Comment:
I tested this one using a nomad plan and I get the following error:

```
rpc error: 1 error occurred: * Task group publishing validation failed: 1 error occurred: * Job type "batch" does not allow update block
```

Deleting this block resolves the issues:

```suggestion
```

Reply:
Ah, toyed with this one (removed other service stuff eg. consul services) but was unsure whether it'd be needed. Glad you've confirmed it isn't. Will remove.